### DISCUSSION TITLE
Possible to use remote file or endpoint for boostrap config?

### DISCUSSION THREAD

**Discussion by staticwave** [directus/directus#13421] [Archive]
Hey! Does anyone know if it's possible to use a remote file or endpoint that returns the config info during bootstrap instead of having it load a config.json locally?

Ideally, I'd like to set CONFIG_PATH to something like https://example.com/config.json before running npx directus bootstrap.

I tried to do this, but it used the default config. Not sure if I'm doing it wrong or it's just not possible.

I'd really appreciate any insights. Thanks!

**Comment by br41nslug** [comment]
The `CONFIG_PATH` setting only supports files in the local file system at this time

**Reply by staticwave** [reply]
Thank you!
